### PR TITLE
Update dockerhub description from github readme.md

### PR DESCRIPTION
**Pull request by xmatthias**
## Summary
Update dockerhub description from the github readme automatically

supplement to #2518 

## Quick changelog

- based on master (there can only be one description on dockerhub, and it should be the latest).

### CODE REVIEW

File: .github/workflows/docker_update_readme.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    branches:
+    - master
+
+jobs:
+  dockerHubDescription:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Docker Hub Description
+      uses: peter-evans/dockerhub-description@v2.0.0
```
Comment (hroff-1902) [inline .github/workflows/docker_update_readme.yml]: `peter-evans/dockerhub-description` says it's v2.1.0 now...

Comment (xmatthias) [reply .github/workflows/docker_update_readme.yml]: huh? this has nothing to do with the raspberry build ... 

this action ONLY updates the dockerhub description - so runtime is ... irrelevant, really (it runs in  a few seconds).

Comment (xmatthias) [reply .github/workflows/docker_update_readme.yml]: nevertheless - updating the action makes sense.

**Comment by xmatthias** [comment]
nope, does not matter - 2 different files .. :)

**Pull request merged**